src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/restfulptv/ServiceChannelsTestsIT.java

```diff
@@ -793,4 +752,5 @@ public void testWebPageChannelNotFound() throws InterruptedException {
     assertNotFound(String.format("/webPageServiceChannels/%s", electronicChannelId));
   }
   
+  // TODO : Service REMOVE tests
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)